runtime/src/main/java/org/corfudb/protocols/wireprotocol/NettyCorfuMessageEncoder.java

```diff
@@ -6,20 +6,36 @@
 
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.concurrent.atomic.LongAccumulator;
+import java.util.function.LongBinaryOperator;
+
 /**
  * Created by mwei on 10/1/15.
  */
 @Slf4j
 public class NettyCorfuMessageEncoder extends MessageToByteEncoder<CorfuMsg> {
 
+
+    final LongAccumulator accumulator = new LongAccumulator(Math::max, Long.MIN_VALUE);
```

Comment:
rename this field to maxValue, as its more descriptive

Reply:
Done.